index.html

```diff
@@ -11,7 +11,8 @@
     <h1>Home Page Directory</h1>
     <div>
       <ul>
-        <li><a href="gale-page.html">Gale's Page</a></li>
```

Comment:
Could you add this line back into the index page as well? Should be able to just copy/paste it back into the file and commit. ❤️